internal/events/event.go

```diff
@@ -139,6 +189,16 @@ func (e *Events) AddMetricsEvent(ctx context.Context, clusterID strfmt.UUID, hos
 	_ = e.saveEvent(ctx, clusterID, hostID, models.EventCategoryMetrics, severity, msg, eventTime, requestID, props...)
```

Comment:
should AddMetricsEvent be extended to support infra-env-id as well ?

Reply:
yup